pkg/proto/configuration/blobstore/blobstore.proto

```diff
@@ -187,6 +187,23 @@ message SingleRedisBlobAccessConfiguration {
 
   // Numerical ID of the database.
   int32 db = 2;
+
+  // Redis Auth Password, "" for NO Password
+  string password = 3;
+
+  // Dial timeout for establishing new connections.
+  // Default is 5 (seconds) or you could select your own seconds '30s'
```

Comment:
Maybe it's best to document it in such a way that the actual default value remains undescribed. It would become invalid if go-redis were to change its default value, right?